dist/pnp.d.ts

```diff
@@ -412,176 +423,1096 @@ declare module "net/httpClient" {
 }
 declare module "sharepoint/rest/mixins" {
     import { Queryable } from "sharepoint/rest/queryable";
+    /**
+     * Implements the $select functionality on classes to which it is applied
+     *
+     */
     export class Selectable extends Queryable {
         select(...selects: string[]): any;
     }
+    /**
+     * Implements the $filter functionality on classes to which it is applied
```

Comment:
Could these be structured in a way that would let us omit comments like these?

Perhaps placing them under an OData namespace/folder/something? Right now the comments just adds noise to the code for me.